src/test/java/core/basesyntax/strategy/impl/PurchaseOperationTest.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax.strategy.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.db.StorageImpl;
+import core.basesyntax.exception.InvalidOperatioExeption;
+import core.basesyntax.model.FruitTransaction;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class PurchaseOperationTest {
+    private static Storage storageImpl;
+    private static PurchaseOperation purchaseOperation;
+
+    @BeforeAll
+    static void setUp() {
+        storageImpl = new StorageImpl();
+        purchaseOperation = new PurchaseOperation(storageImpl);
+    }
+
+    @Test
+    void process_enoughQuantity_successfullyUpdatedStorage() {
+        Storage.storage.put("banana", 100);
```

Comment:
As Arthur suggested please use `storageImple` instance instead of `Storage.storage` for consistency. 
Fix here and in all similar case.